examples/fuse_examples/imaging/classification/mnist/simple_mnist_starter.py

```diff
@@ -0,0 +1,164 @@
+"""
+(C) Copyright 2021 IBM Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Created on June 30, 2021
+
+===============================
+
+The most minimal MNIST classifier implementation that demonstrate end to end training using Fuse that we could make.
+There is a lot more that one could do on top of this, including inference, evaluation, model checkpointing, etc. But
+the goal of this script is simply to train a classifier and show how to wrap a regular pytorch model so it can be
+trained "fuse" style. Meaning it accesses data using keys to a batch_dict. It also demonstrates how easy it is to
+load datasets from fuse.
+"""
+
+import copy
+import os
+from typing import OrderedDict
+
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data.dataloader import DataLoader
+import pytorch_lightning as pl
+
+
+from fuse.eval.metrics.classification.metrics_thresholding_common import MetricApplyThresholds
+from fuse.eval.metrics.classification.metrics_classification_common import MetricAccuracy
+
+from fuse.data.utils.samplers import BatchSamplerDefault
+from fuse.data.utils.collates import CollateDefault
+
+from fuse.dl.losses.loss_default import LossDefault
+from fuse.dl.models.model_wrapper import ModelWrapSeqToDict
+from fuse.dl.lightning.pl_module import LightningModuleDefault
+
+from fuseimg.datasets.mnist import MNIST
+from examples.fuse_examples.imaging.classification.mnist import lenet
+
+## Paths and Hyperparameters ############################################
+ROOT = "_examples/mnist"  # TODO: fill path here
+MODEL_DIR = os.path.join(ROOT, "model_dir")
+PATHS = {
+    "model_dir": MODEL_DIR,
+    "cache_dir": os.path.join(ROOT, "cache_dir"),
+    "inference_dir": os.path.join(MODEL_DIR, "infer_dir"),
```

Comment:
You are not using inference dir and eval dir, let's delete it.

Reply:
done :)